plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -177,6 +177,34 @@ func PodMatchesNodeLabels(pod *api.Pod, node *api.Node) bool {
 	return selector.Matches(labels.Set(node.Labels))
 }
 
+// NoPodLabelConflict evaluates if a pod fits a node by:
+// 1) Comparing the pod's conflict labels against labels of all the other pods.
+// 2) Comparing all the other pods conflict labels with labels of this pod.
+func NoPodLabelConflict(pod *api.Pod, existingPods []*api.Pod, node string) (bool, error) {
+	for podIx := range existingPods {
+		if existingPods[podIx].Labels != nil {
+			for conflict, value := range pod.Spec.Conflicts {
+				if existingValue, ok := existingPods[podIx].Labels[conflict]; ok {
```

Comment:
@bgrant0607 

When I tried to use []labels.LabelSelector, I ran in to this issue with generated conversion code.

pkg/api/v1/conversion_generated.go:1535: cannot use make([]labels.LabelSelector, len(in.Conflicts)) (type []labels.LabelSelector) as type [][]labels.Requirement in assignment

Perhaps someone familiar with the code generation can comment on whether slice of slice is ok.

I changed it to [][]labels.Requirement for now and submitted a patch.